src/bootstrap-wysiwyg.js

```diff
@@ -205,13 +206,18 @@
 				editor.focus();
 				$.each(files, function (idx, fileInfo) {
 					if (/^image\//.test(fileInfo.type)) {
-						$.when(readFileIntoDataUrl(fileInfo)).done(function (dataUrl) {
-							underscoreThrottle(execCommand('insertimage', dataUrl), options.keypressTimeout);
-							editor.trigger('image-inserted');
-						}).fail(function (e) {
-							options.fileUploadError("file-reader", e);
-						});
-					} else {
+						if (fileInfo.size>maxFileSize){
```

Comment:
I'd be happier if this was testing for the presence of a config first, rather than just assuming there is one. If we rewrite this to remove the absolute dependency on a file limit, I think we can introduce it.